utils/json.c

```diff
@@ -217,6 +217,27 @@ char *pv_json_format(const char *buf, int len)
 	return json_string;
 }
 
+bool pv_json_is_valid(const char *json)
+{
+	bool ret = false;
+
+	int tokc;
+	jsmntok_t *tokv = NULL;
+
+	if (jsmnutil_parse_json(json, &tokv, &tokc) < 0)
+		goto out;
+
+	if (!jsmnutil_get_object_keys(json, tokv))
```

Comment:
This second check is needed? and is our parser running in strict format? see (https://github.com/zserge/jsmn/blob/master/jsmn.h#L145)

Reply:
Thanks for the comment. I'll switch that to jsmnutil_object_key_count, which only checks if first token is JSMN_OBJECT. Apparently, jsmnutil_parse_json will take any string as good and just set it as JSMN_PRIMITIVE.